docs/troubleshooting.md

```diff
@@ -1,9 +1,21 @@
 # Troubleshooting
 
-## Handling of assembly version conflicts
+## Assembly version conflicts
 
 OpenTelemetry .NET NuGet packages and its dependencies
 are deployed with the OpenTelemetry .NET Automatic Instrumentation.
+
+In case of assembly version conflicts you may get a `TargetInvocationException`.
```

Comment:
I think on .NET Framework the behavior could be different (no time to test it at the moment).